boards/posix/native_posix/main.c

```diff
@@ -134,6 +135,15 @@ int main(int argc, char *argv[])
 const uint8_t *posix_fuzz_buf;
 size_t posix_fuzz_sz;
 
+static int posix_fuzz_result;
```

Comment:
I'd much rather go with a `native_` prefix instead of `posix_` both for variables and functions.

Reply:
Should I rename posix_fuzz_buf and posix_fuzz_sz as well?

Reply:
That would be nice. But maybe lets do that then in the follow up PR.